src/SuiteSparseMatrixCollection.jl

```diff
@@ -114,8 +114,12 @@ function fetch_ssmc(matrices; format="MM")
       download(url, fname)
     end
     if ext == "tar.gz"
-      tar_options = Sys.iswindows() ? "--force-local" : ""
-      run(`tar $tar_options -zxf $fname -C $g_path`)
+      if Sys.iswindows()
+        exe7z = isdefined(Base, :LIBEXECDIR) ? joinpath(Sys.BINDIR, Base.LIBEXECDIR, "7z.exe") : joinpath(Sys.BINDIR, "7z.exe")
+        pipeline(`$exe7z x $fname -y -so`, `$exe7z x -si -y -ttar -o$g_path`)
+      else
+        run(`tar -zxf $fname -C $g_path`)
+      end
```

Comment:
Let's not do this either. It looks like it's the tar file that has some issues, not the tar command. I'll see if I can make those matrices available as artifacts instead.